transformation/src/test/test-files/outputs/donor/part-1.json

```diff
@@ -0,0 +1,4 @@
+{"age_max":19,"age_min":16,"age_unit":"week","award":"/awards/U41HG006992/","cross_references":[],"id":"ENCDO273AAA","lab":"/labs/peggy-farnham/","life_stage":"embryonic","organism":"/organisms/human/","parent_donor_ids":[],"submitted_by":"/users/4c23ec32-c7c8-4ac0-affb-04befcc881d4/","time_created":"2013-09-13T13:28:25.742869-08:00"}
+{"age_max":37,"age_min":37,"age_unit":"year","award":"/awards/ENCODE/","cross_references":["GEO:SAMN05897796"],"id":"ENCDO845WKR","lab":"/labs/encode-consortium/","life_stage":"adult","organism":"/organisms/human/","parent_donor_ids":[],"sex":"male","submitted_by":"/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/","time_created":"2015-02-14T00:31:58.410757Z","twin_donor_id":"ENCDO241AAA"}
+{"award":"/awards/U41HG006992/","cross_references":["GEO:SAMN04284513"],"ethnicity":"Caucasian","id":"ENCDO000AAK","lab":"/labs/encode-consortium/","life_stage":"adult","organism":"/organisms/human/","parent_donor_ids":["ENCDO000ACA","ENCDO000AAL"],"sex":"female","submitted_by":"/users/3787a0ac-f13a-40fc-a524-69628b04cd59/","time_created":"2013-10-24T03:45:46.823758Z"}
+{"award":"/awards/U41HG006992/","cross_references":[],"id":"ENCDO241AAA","lab":"/labs/john-stamatoyannopoulos/","life_stage":"newborn","organism":"/organisms/human/","parent_donor_ids":[],"sex":"male","submitted_by":"/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/","time_created":"2013-09-13T13:28:29.344052-08:00","twin_donor_id":"ENCDO845WKR"}
```

Comment:
These antibody and donor output files already exist, right? Although it looks like they might be currently be misnamed (`part1.json` instead of `part-1.json`). I just want to make sure that we delete (or rename) the old ones.